File: Python/Arreglos/Arreglos-3.py
cantNotasString = input("Ingrese el tamano del arreglo: ")

cantNotas = int(cantNotasString)
notas = []
suma = 0
for i in range(0, cantNotas):
    notas.append(float(input("Ingrese el numero " + str(i + 1) + ": ")))

for i in notas:
    suma = i + suma

promedio = suma / len(notas)
print("El promedio es: ", promedio)

File: CPP/Arreglos/Arreglos-3.cpp
#include <iostream>
#include <stdio.h>
using namespace std;
int main() {
 int catNumeros = 0;
 int numeros[catNumeros];
 float suma = 0;
 float promedio;
 cout<<"Ingrese tamaño del arreglo: ";
 cin>>catNumeros;
 for (int i = 0; i < catNumeros; i++){
   //numeros[i] = i + 1;
   cout<<"Ingrese el dato: "<<(i + 1)<<endl;
   cin>>numeros[i];
 }
  // mostrar contenido del arreglo

 for (int i = 0; i < catNumeros; i++){
   suma = numeros[i]+suma;
 }
 promedio = suma / catNumeros;
 cout<<"El promedio es: "<< promedio<<endl;
 }
